build/build.proj

```diff
@@ -415,4 +415,28 @@
     <Message Text="Desktop $(TestResultsFileName) tests passed! Results: $(TestResultsXunit)" Condition=" '$(SkipDesktopAssemblies)' != 'true' AND '$(DesktopTestErrorCode)' == '0' " Importance="High" />
     <Message Text="NETCore $(TestResultsFileName) tests passed! Results: $(TestResultsVsts)" Condition=" '$(SkipCoreAssemblies)' != 'true' AND '$(VSTestErrorCode)' == '0' " Importance="High" />
   </Target>
+
+
+  <!--
+    ============================================================
+    Generate documentation for source projects
+    ============================================================
+  -->
+  <UsingTask AssemblyFile="$(RepositoryRootDirectory)\tools-local\doc.tasks\bin\Debug\netstandard2.0\doc.tasks.dll" TaskName="NuGetTasks.GenerateMarkdownDoc" />
+  <Target Name="GenerateDocs">
+    <MsBuild
+      Projects="$(RepositoryRootDirectory)\tools-local\doc.tasks\doc.tasks.csproj"
+      Targets="Restore;Build"
```

Comment:
Don't do this. 
See: https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#restoring-and-building-with-one-msbuild-command.

Reply:
Changed to two MSBuild task invocations